cli/fs.rs

```diff
@@ -95,7 +96,7 @@ pub fn make_temp(
 }
 
 pub fn mkdir(path: &Path, perm: u32, recursive: bool) -> std::io::Result<()> {
-  debug!("mkdir -p {}", path.display());
+  // debug!("mkdir -p {}", path.display());
```

Comment:
Unnecessary. You can either remove the line or leave it as is, but leaving dead code isn't helpful. Suggest leaving it as is.